tests/test_integration.py

```diff
@@ -0,0 +1,157 @@
+import dash
+from dash import html, Input, Output, State
+from dash_player import DashPlayer
+
+
+def test_001_child_with_0(dash_duo):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(id="nully-wrapper", children=0)
+    dash_duo.start_server(app)
+    dash_duo.wait_for_text_to_equal("#nully-wrapper", "0", timeout=4)
+
+    assert dash_duo.find_element("#nully-wrapper").text == "0"
+    assert dash_duo.get_logs() == [], "browser console should contain no error"
+    dash_duo.percy_snapshot("test_001_child_with_0-layout")
+
+
+def test_002_dash_player_exists_with_size(dash_duo):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        children=[
+            DashPlayer(
+                id="video-player",
+                url="https://media.w3.org/2010/05/bunny/movie.ogv",
+                playing=True,
+                controls=True,
+                width="640px",
+                height="360px",
+            )
+        ]
+    )
+    dash_duo.start_server(app)
+    assert (
+        dash_duo.find_element("#video-player").get_attribute("style")
+        == "width: 640px; height: 360px;"
+    )
+    assert (
+        dash_duo.find_element("#video-player > video").get_attribute("src")
+        == "https://media.w3.org/2010/05/bunny/movie.ogv"
+    )
+    assert dash_duo.get_logs() == [], "browser console should contain no error"
+    dash_duo.percy_snapshot("test_002_dash_player_exists-layout")
```

Comment:
Probably don't need to bring percy into these tests?

Reply:
Removed